### PR TITLE
Add information on VFX platform support to repo

### PR DESCRIPTION
**Pull request by ssteinbach**
- adds notes to README.md and the contributing doc file about what our policy is, as well as links to the VFX platform homepage explaining it.

### CODE REVIEW

**Review by meshula — APPROVED** [review]
Lgtm

File: README.md [hunk 1]
```diff
@@ -48,6 +48,16 @@ Documentation
 --------------
 Documentation, including quick start, architecture, use cases, API docs, and much more, is available on [ReadTheDocs](https://opentimelineio.readthedocs.io/)
 
+Support Platforms
```
Comment (JeanChristopheMorinPerso) [inline README.md:51]: Should this be `Supported Platforms`? Support platforms almost looks like it's about where can users get support (slack, GitHub, etc).

Comment (ssteinbach) [reply README.md:51]: how about `Supported VFX Platforms` instead?  Since its really just talking about vfx platform.

Comment (JeanChristopheMorinPerso) [reply README.md:51]: Even better!

**Review by jminor — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (meshula) [reply README.md:51]:
```suggestion
Supported VFX Platforms
```

**Comment by meshula** [comment]
I amended the commit with "Supported VFX Platforms" per the review conversation

**Pull request merged**